examples/pytorch_mc_dropout.py

```diff
@@ -0,0 +1,126 @@
+"""
+In this file the basic ModAL PyTorch DeepActiveLearner workflow is explained 
+through an example on the MNIST dataset and the MC-Dropout-Bald query strategy.
+"""
+import sys
+import os
+import torch
+from torch import nn
+from skorch import NeuralNetClassifier
+
+from modAL.models import DeepActiveLearner
+
+# import of query strategies
+from modAL.dropout import mc_dropout_bald, mc_dropout_mean_st
+
+import numpy as np
+from torch.utils.data import DataLoader
+from torchvision.transforms import ToTensor
+from torchvision.datasets import MNIST
+
+# Standard Pytorch Model (Visit the PyTorch documentation for more details)
+class Torch_Model(nn.Module):
+    def __init__(self,):
+        super(Torch_Model, self).__init__()
+        self.convs = nn.Sequential(
+            nn.Conv2d(1, 32, 3),
+            nn.ReLU(),
+            nn.Conv2d(32, 64, 3),
+            nn.ReLU(),
+            nn.MaxPool2d(2),
+            nn.Dropout(0.25)
+        )
+        self.fcs = nn.Sequential(
+            nn.Linear(12*12*64, 128),
+            nn.ReLU(),
+            nn.Dropout(0.5),
+            nn.Linear(128, 10),
+        )
+
+    def forward(self, x):
+        out = x
+        out = self.convs(out)
+        out = out.view(-1, 12*12*64)
+        out = self.fcs(out)
+        return out
+
+
+torch_model = Torch_Model()
+"""
+You can acquire from the layer_list the dropout_layer_indexes, which can then be passed on 
```

Comment:
Hello @cosmic-cortex,

Would you mind to answer, active learning framework is valid only if the model is probabilistic? Why do you choose to work with dropout to focus on?

Reply:
if you use dropout at prediction time and do several predictions, it will give different output probabilities, it can be seen as an approximation of a deep Bayesian network (see this paper https://arxiv.org/abs/1506.02142)